tasks/main.yml

```diff
@@ -1 +1,31 @@
 # role tasks
+
+- name: Install necessary packages
+  package:
+    name: parted
+    state: present
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+
+- name: Run wipefs to clear the disk signatures
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: "wipefs --all --force {{ item }}"
+  register: signatures_before
+  changed_when: true
```

Comment:
This is redundant. The `command` module should always report a status of "changed" when it is executed without a `changed_when` directive or a `creates` or `removes` argument.

Reply:
By default I am passing in an empty list of disks.   Without the changed when command travis/ci will not pass with the following error.
Commands should not change things if nothing needs doing

Reply:
In this case you should flag the linter to ignore that warning.